doc/community-tools.md

```diff
@@ -0,0 +1,13 @@
+#Community tools
+
+This section provides a list of community-developed tools that integrate with cardano-db-sync. We appreciate the efforts of these developers in expanding the ecosystem. If you have a tool you'd like to add to this list, please open an issue or a pull request.
+
+### Koios
+- **Description:** The Koios Artifacts repository is part of the Cardano community and acts as a collection or set of artifacts for the Koios service. Koios is an open-source, lightweight Cardano REST API that provides various endpoints to interact with the Cardano blockchain. 
+- **Repository:** [https://github.com/cardano-community/koios-artifacts](https://github.com/cardano-community/koios-artifacts)
```

Comment:
My suggestion for an on-looker who may not be interested in entire repository would be to link directly to folders where SQLs are located (i.e. https://github.com/cardano-community/koios-artifacts/tree/main/files/grest/rpc) for Koios

Reply:
I've added those direct links in `interesting-queries.md`. This file is more about the whole project. Though I could also add the SQL queries here too so it's in more than one place.